QuantConnect.BybitBrokerage/Api/BybitPositionApiEndpoint.cs

```diff
@@ -49,10 +49,33 @@ public IEnumerable<BybitPositionInfo> GetPositions(BybitProductCategory category
     {
         if (category == BybitProductCategory.Spot) return Array.Empty<BybitPositionInfo>();
 
-        var parameters = new KeyValuePair<string, string>[]
+        var parameters = new List<KeyValuePair<string, string>>();
+        
+        if (category == BybitProductCategory.Linear)
         {
-            new("settleCoin", "USDT")
-        };
+            parameters.Add(KeyValuePair.Create("settleCoin", "USDT"));
+        }
+
         return FetchAll<BybitPositionInfo>("/position/list", category, 200, parameters, true);
     }
+
+    /// <summary>
+    /// It supports to switch the position mode for USDT perpetual and Inverse futures.
+    /// If you are in one-way Mode, you can only open one position on Buy or Sell side. If you are in hedge mode, you can open both Buy and Sell side positions simultaneously.
+    /// </summary>
+    /// <param name="category">The product category</param>
+    /// <param name="symbol">The symbol for which the mode should be changed</param>
+    /// <param name="mode">The mode which should be set</param>
+    public void SwitchPositionMode(BybitProductCategory category, Symbol symbol, PositionMode mode)
```

Comment:
How/where is this going to be used?

Reply:
It's not used but the question is if it should. Hedge mode is currently not supported and for inverse the default mode is hege which can only be changed to normal mode by using the API. 
So should this be done automatically or is the user responsible for that, if so is there a way to make this function available to the user in any way. It would definitely need to be documented.